### PR TITLE
Warn when number of columns returned doesn't match number of fields provided

### PR DESCRIPTION
**Pull request by abelcastilloavant**
When making a query, the expectation is to receive one column per field provided. This PR adds a warning if that expectation is not satisfied.

I considered giving an error instead of a warning, but I want the user to receive data - e.g. if the user makes an expensive query, I'd like the user to at least have a chance to inspect the columns actually pulled.

### CODE REVIEW

File: R/run_inline_query.R [hunk 1]
```diff
@@ -18,9 +18,7 @@ run_inline_query <- function(base_url, client_id, client_secret,
                       limit = 1000, silent_read_csv) {
 
 
-  # The API requires you to "log in" and obtain a session token
-  # TODO: find a way to cache the session token, perhaps using memoise package?
```
Comment (abelcastilloavant) [inline R/run_inline_query.R]: removing obsolete TODO - this was already done.

**Comment by peterhurford** [comment]
LGTM :+1: :shipit:

File: R/run_inline_query.R [hunk 2]
```diff
@@ -29,5 +27,12 @@ run_inline_query <- function(base_url, client_id, client_secret,
   inline_query_response <- query_api_call(base_url,
     model, view, fields, filters, limit) 
 
-  extract_query_result(inline_query_response, silent_read_csv)
+  output <- extract_query_result(inline_query_response, silent_read_csv)
+
+  if (ncol(output) != length(fields)) {
+    warning(paste("Looker data has", ncol(output), "columns,",
+            "but", length(fields), "fields provided."))
```
Comment (peterhurford) [inline R/run_inline_query.R]: You could potentially tell them which fields didn't make it.

Comment (abelcastilloavant) [inline R/run_inline_query.R]: Thought about it, but format of output names  - best bet is to report the column names of the output and the fields, and hope the user can figure out which fields didn't make it?

**Comment by khalidharun** [comment]
## [Current coverage][cc-pull] is **66.66%**
> Merging [#32][cc-pull] into [master][cc-base-branch] will decrease coverage by **9.52%**

```diff
@@             master        #32   diff @@
==========================================
  Files             6          5     -1   
  Lines            63         72     +9   
  Methods           0          0          
  Messages          0          0          
  Branches          0          0          
==========================================
  Hits             48         48          
- Misses           15         24     +9   
  Partials          0          0          
```

> Powered by [Codecov](https://codecov.io?src=pr). Last updated by [95ebe87...f0aef0a][cc-compare]
[cc-base-branch]: https://codecov.io/gh/avantcredit/looker3/branch/master?src=pr
[cc-compare]: https://codecov.io/gh/avantcredit/looker3/compare/95ebe87f4864ad04bcdb236850b3c71af801a9e7...f0aef0a8a1e7de9cdb62b896d859be8cde61a3de
[cc-pull]: https://codecov.io/gh/avantcredit/looker3/pull/32?src=pr

**Comment by abelcastilloavant** [comment]
Made the warning give the counts and the names.

**Comment by peterhurford** [comment]
LGTM :+1: :shipit:

**Comment by abelcastilloavant** [comment]
tagged and released 0.1.13

**Pull request merged**